tests/test_strategy_load_hours.py

```diff
@@ -41,6 +41,13 @@
 MIN_BUY_ENERGY = 50  # wh
 
 
+@pytest.fixture(scope="function", autouse=True)
+def auto_fixture():
+    GlobalConfig.sim_duration = duration(days=1)
+    GlobalConfig.slot_length = duration(minutes=15)
```

Comment:
As above: shouldn't we provide a teardown step after the `yield` to restore the default values?

Reply:
Similar to the former comment, found out the root cause and the setup method should not be needed for this module